apps/o2-blocks/package.json

```diff
@@ -19,7 +19,7 @@
 	"scripts": {
 		"clean": "npx rimraf dist",
 		"prebuild": "npm run clean",
-		"build": "webpack --config='../../packages/calypso-build/webpack.config.js' --output-path='./dist' --env.WP='true' editor='./src/editor.js' "
+		"build": "cb --output-path='./dist' editor='./src/editor.js' --env.WP"
```

Comment:
Will be nice to drop `--output-path` as well https://github.com/Automattic/wp-calypso/pull/32466/files#r277540759

Reply:
:cherries: picked: bd0cb1837b
Dropped `output-path` arg per 56f9398614